env-builder-data/build/script/packet/mlt-6.6.0.sh

```diff
@@ -45,6 +45,10 @@ pkbuild() {
         return 1
     fi
 
+    PK_CFLAGS="-I/usr/$HOST/include/"
```

Comment:
I think it's better to extract this changes in a separate PR.

Reply:
Done! - https://github.com/morevnaproject/morevna-builds/pull/81